package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "dev": "jspm install && python -m SimpleHTTPServer",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "karma start karma.conf.js"
```

Comment:
let's actually point that at the local karma. `node ./node_modules/karma/bin/karma start karma.conf.js`